libsqlite3-sys/build.rs

```diff
@@ -239,7 +239,9 @@ mod build_bundled {
         if !win_target() {
             cfg.flag("-DHAVE_LOCALTIME_R");
         }
-        if env::var("TARGET").map_or(false, |v| v.starts_with("wasm32-wasi")) {
+        if env::var("TARGET").map_or(false, |v| {
+            v.starts_with("wasm32-wasi") || v.starts_with("wasm32-wasip1")
```

Comment:
`wasm32-wasip1` starts with `wasm32-wasi` !
See #1569

Reply:
Oops! Sorry, I got it wrong, and "fixed" the non-existing bug! 
Looking forward to seeing the new release.